packages/client/src/main.js

```diff
@@ -23,6 +23,10 @@ if (window.APP_CONFIG?.GOOGLE_TAG_ID) {
   store.watch((state) => state.users.loggedInUser, (newUser) => setUserForGoogleAnalytics(newUser));
 }
 
+store.watch((state) => state.users.loggedInUser, (newUser) => datadogRum.setUser({
+  id: newUser.id, 'agency.id': newUser.agency_id, role: newUser.role.name, 'organization.id': newUser.tenant_id,
```

Comment:
This might have been a lack of clarity on my part (I think I provided an example where `.id` was intended to convey nested objects), but I think using the `_id` convention will be easier to work with. 

```suggestion
  id: newUser.id, agency_id: newUser.agency_id, role: newUser.role.name, organization_id: newUser.tenant_id,
```